src/ubx.cpp

```diff
@@ -2038,6 +2038,12 @@ GPSDriverUBX::payloadRxDone()
 			bool rel_pos_valid = _buf.payload_rx_nav_relposned.flags & (1 << 2);
 			(void)rel_length_acc;
 
+			// RTK float fix type is not accurate enough 
```

Comment:
Instead of depending on the fix_type being set I would directly check the `carrier_solution_floating` flag in NAV-RELPOSNED here.

See the carrier_solution_floating and carrier_solution_fixed handling below.

Reply:
I've updated the code, please let me know if this is what you meant @dagar

Reply:
I would roll carrier_solution_fixed right into the conditional below, but otherwise this looks good.